templates/managed-ec2-v5.yaml

```diff
@@ -72,7 +72,6 @@ Parameters:
       - c5.9xlarge
       - c5.18xlarge
       - g2.2xlarge
-      - g2.8xlarge
```

Comment:
Just curious:  Why remove g2.8xlarge instances?

Reply:
good catch,  i just noticed that nothing depends on `managed-ec2-v5.yaml` template anymore so I just removed it.